backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/optimizations/DFGSerializer.kt

```diff
@@ -947,7 +947,7 @@ internal object DFGSerializer {
         val publicFunctionsMap = mutableMapOf<Long, DataFlowIR.FunctionSymbol.Public>()
         val functions = mutableMapOf<DataFlowIR.FunctionSymbol, DataFlowIR.Function>()
         val specifics = context.config.configuration.get(CommonConfigurationKeys.LANGUAGE_VERSION_SETTINGS)!!
-        val storageManager = LockBasedStorageManager()
+        val storageManager = LockBasedStorageManager("deserializer-$startPrivateFunIndex-$startPrivateFunIndex")
```

Comment:
I think there is a misprint, it should be `"deserializer-$startPrivateTypeIndex-$startPrivateFunIndex"` actually

Reply:
done